src/main/resources/db/changelog/2024-05-23-update-primary-contact-paradigm-in-households.yaml

```diff
@@ -11,9 +11,9 @@ databaseChangeLog:
             sql: ALTER TABLE account ADD COLUMN `is_primary_contact_for_household` bit(1) DEFAULT NULL AFTER `household_id`;
         - sql:
             dbms: 'h2, mysql'
-            sql: UPDATE account a 
-              JOIN household h ON a.household_id = h.id 
-              SET a.is_primary_contact_for_household = 1 
+            sql: UPDATE account a
```

Comment:
Sometimes code cleanup gets a little overzealous. This could cause checksum issues in the upper environments. We can setup the code reformatter to exclude the `*.yaml` files.